src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
+    protected Color color;

```

Comment:
let's create a public constructor that accepts color - in order to set color from child figures' constructors

and also let's create a public getter

Reply:
Not completely fixed. Create `getColor()` method that should return value of color variable